nilearn/datasets/utils.py

```diff
@@ -617,11 +617,12 @@ def _fetch_files(data_dir, files, resume=True, mock=False, verbose=1):
         Unique dataset name
 
     files: list of (string, string, dict)
-        List of files and their corresponding url. The dictionary contains
-        options regarding the files. Options supported are 'uncompress' to
-        indicates that the file is an archive, 'md5sum' to check the md5 sum of
-        the file and 'move' if renaming the file or moving it to a subfolder is
-        needed.
+        List of files and their corresponding url with dictionary that contains
+        options regarding the files. Ex. (file_name, url, opt). If a file_name
```

Comment:
We use "eg" instead of "ex" as in `the files, eg (...)`. Also, I would say `file_path` instead of `file_name` ot underline the fact that `subfolder/file_name` is a valid value.